tests/testthat/test-batch_downloader.R

```diff
@@ -0,0 +1,157 @@
+test_that("batch_downloader works as expected", {
+  skip("batch_downloader requires API call")
+
+  api_call_n <- paste0(url, "api/outbreaks/", outbreak_id, "/cases/count")
+  api_call_get <- paste0(url, "api/outbreaks/", outbreak_id, "/cases")
+  res <- batch_downloader(
+    url = url,
+    username = username,
+    password = password,
+    api_call_n = api_call_n,
+    api_call_get = api_call_get,
+    batch_size = 50000
+  )
+
+
+  expect_s3_class(res, "tbl_df")
+  expect_s3_class(res, "data.frame")
+  expect_identical(dim(res), c(13L, 153L))
```

Comment:
Also fails when using the outbreak from `get_active_outbreak()` - 14 rows returned instead of 13.